### DISCUSSION TITLE
10% Early Bird discount for 4-day Security Onion 2 Fundamentals for Analysts & Admins Training Class

### DISCUSSION THREAD

**Discussion by dougburks** [Security-Onion-Solutions/securityonion#7657] [Announcements]
https://blog.securityonion.net/2022/03/10-early-bird-discount-for-4-day_28.html

**Comment by vanh20** [comment]
Hello Doug,

   Can you please post here or send me a link on how I can benefit most out of this training. I really want to come in prepared and possibly, have some backup in-case something untoward happens. My system has passed the pre-check requirements.

   If this is not the appropriate forum to request such, please accept my apologies. It's only a week from now and I'm a bit excited and  panicking at the same time. :D
   
Thank you for this great product and training opportunity. :)

**Reply by dougburks** [reply]
Looks like you also sent a message to our training team via the Eventbrite page. Our training manager has responded to your email.

**Reply by vanh20** [reply]
Yes. They did, and I apologize. I didn't quite expect they would respond almost in a flash. They answered my technical concerns quite well.

Thank you! :)
